trends_data/src/commonMain/kotlin/net/thechance/mena/trends/data/dto/ProfileDto.kt

```diff
@@ -0,0 +1,18 @@
+package net.thechance.mena.trends.data.dto
+
+
+import kotlinx.serialization.SerialName
+
+import kotlinx.serialization.Serializable
+@Serializable
+data class ProfileDto(
```

Comment:
and i think if we make it 
```suggestion
internal data class ProfileDto(